core/src/main/java/bisq/core/user/Preferences.java

```diff
@@ -98,7 +98,8 @@
             new BlockChainExplorer("SoChain. Wow.", "https://chain.so/tx/BTC/", "https://chain.so/address/BTC/"),
             new BlockChainExplorer("Blockchain.info", "https://blockchain.info/tx/", "https://blockchain.info/address/"),
             new BlockChainExplorer("Insight", "https://insight.bitpay.com/tx/", "https://insight.bitpay.com/address/"),
-            new BlockChainExplorer("Blockchair", "https://blockchair.com/bitcoin/transaction/", "https://blockchair.com/bitcoin/address/")
+            new BlockChainExplorer("Blockchair", "https://blockchair.com/bitcoin/transaction/", "https://blockchair.com/bitcoin/address/"),
+            new BlockChainExplorer("Mynode.local", "http://mynode.local:3002/tx/", "http://mynode.local:3002/address/")
```

Comment:
How about calling it "Local BTC RPC Explorer"?

"BTC RPC Explorer" is the official name of the explorer project: https://github.com/janoside/btc-rpc-explorer

Reply:
But no, its fine, please disregard that, its only a cosmetic thing.